friendshipper/src/routes/+layout.svelte

```diff
@@ -277,8 +281,12 @@
 			try {
 				await tryOktaRefresh();
 			} catch (_) {
-				await handleOktaLogout();
-				await handleOktaLogin();
+				try {
```

Comment:
oh the nested try catch

so, if refresh fails, then do a logout and login, nice